<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gB_AvailablePlayers">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>40</y>
      <width>250</width>
      <height>800</height>
     </rect>
    </property>
    <property name="title">
     <string>List of available users</string>
    </property>
    <widget class="QTableWidget" name="tW_availablePlayers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>230</width>
       <height>711</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>115</number>
     </attribute>
    </widget>
    <widget class="QPushButton" name="pB_requestNewGame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>750</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Request New Game</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pB_logout">
    <property name="geometry">
     <rect>
      <x>1660</x>
      <y>10</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gB_board">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1641</width>
      <height>841</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pB_resign">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>30</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Resign</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_time">
     <property name="geometry">
      <rect>
       <x>1500</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>60</y>
       <width>1611</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
